src/test_files/discordTest.js

```diff
@@ -0,0 +1,12 @@
+const DiscordBot = require('../../discordBot.js')
+
+
+var testBot = new DiscordBot(process.env.DISCORD_BOT_TOKEN);
+testBot.start().then((status) => {
+  console.log(status);
+})
+testBot.sendFile('source.gif','714930494126424227');
+testBot.sendGoogleForm('https://forms.gle/Beyb8EuojwJwqXTw9', '714930494126424227');
+testBot.listenForGoogleFormSubmissions(3002, (answers) => {
+  console.log(answers);
+});
```

Comment:
I've setup the mocha test framework, we could put the test file under the /test/, so they could run `npm test` to do the unit test. Also there's some `(node:9967) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'send' of undefined` error while I'm trying running the test, it seems you need to put your function call in the promise chain